pkg/deploy/generator/scripts/util-services.sh

```diff
@@ -286,17 +315,232 @@ ExecStart=/usr/bin/podman run \
   -v /run/systemd/journal:/run/systemd/journal \
   -v /var/etw:/var/etw:z \
   -v /var/run/mdsd/asa:/var/run/mdsd/asa:z \
-  $image \
+  $RPIMAGE \
   portal
 Restart=always
 RestartSec=1
 
 [Install]
-WantedBy=multi-user.target"
+WantedBy=multi-user.target'
 
     write_file aro_portal_service_filename aro_portal_service_file true
 }
 
+# configure_service_aro_mise
+# args:
+# 1) image - nameref, string; MISE container image
+# 2) ipaddress - nameref, string; static ip of podman network to be attached
+configure_service_aro_mise() {
+    local -n image="$1"
+    local -n ipaddress="$2"
+    log "starting"
+    log "Configuring aro-mise service"
+
+    LOGININSTANCE="https://login.microsoftonline.com"
+    if [[ $AZURECLOUDNAME == "AzureUSGovernment" ]]; then
```

Comment:
Can you add `AzureUSGovernment` as a constant (read only var `declare -r`) in `util-common.sh`?
We may want to check the cloud instance in other places, and it would be good to set it as a read only variable now.

Reply:
Done. added this is constant and addressed here accordingly.